### PR TITLE
Create header logo in portal page

### PR DESCRIPTION
**Pull request by jayushi-canyon**
### Ticket(s) or GitHub Issue

- https://canyongbs.atlassian.net/browse/AIDAPP-108

### Technical Description

> In portal section add an media field to upload header logo.
> Create migration for field and also doe pennant/feature.

### CODE REVIEW

File: app-modules/portal/src/Filament/Pages/ManagePortalSettings.php [hunk 1]
```diff
@@ -170,7 +173,19 @@ public function form(Form $form): Form
                             )
                             ->columnSpanFull(),
                     ])->columns(2),
-
+                Section::make('Header')
+                    ->schema([
+                        SpatieMediaLibraryFileUpload::make('logo')
+                            ->disk('s3')
```
Comment (mxm1070) [inline app-modules/portal/src/Filament/Pages/ManagePortalSettings.php:179]: I know we've done this in a bunch of places previously, but to help future clean up please remove setting the disk here so we can rely on the `FILAMENT_FILESYSTEM_DISK` env key to handle the disk instead.

**Review by mxm1070 — CHANGES_REQUESTED** [review]
Are you intending to do the frontend work in a separate pr?

**Review by Orrison — CHANGES_REQUESTED** [review]
Looks good, let's move forward with @mxm1070's suggestions before merging though.

File: app-modules/portal/src/Filament/Pages/ManagePortalSettings.php (hunk 1 above)
Comment (jayushi-canyon) [reply app-modules/portal/src/Filament/Pages/ManagePortalSettings.php:179]: @mxm1070 Correct me if I am wrong, so you are expecting changes as below?
->disk(env(FILAMENT_FILESYSTEM_DISK), 's3' )
And in env set FILAMENT_FILESYSTEM_DISK=s3

Comment (Orrison) [reply app-modules/portal/src/Filament/Pages/ManagePortalSettings.php:179]: @jayushi-canyon, no. You can just remove the `disk` method here. As Filament will just default to the default system disk. Which we currently have set to `s3`.

Comment (jayushi-canyon) [reply app-modules/portal/src/Filament/Pages/ManagePortalSettings.php]: @mxm1070 @Orrison Made the required changes.

**Review by Orrison — APPROVED** [review]

**Pull request merged**